safe_transaction_service/utils/tests/test_tasks.py

```diff
@@ -1,16 +1,34 @@
+from unittest import mock
```

Comment:
What is the context for these changes? 😅

Reply:
Oh that's right, those changes were supposed to go on the increase coverage PR. I realised some of the tasks were not tested at all

Reply:
Removed